### PR TITLE
Misc queue from 2018 (sorry, lost sources) 

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -8339,22 +8339,32 @@ ivocation->invocation
 iwithout->without
 iwll->will
 iwth->with
+jagwar->jaguar
+jalusey->jalousie
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: While it turns out this is a word ( https://en.wikipedia.org/wiki/Jalousie_window ), the main suggestion should probably be jealousy.

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -8339,22 +8339,32 @@ ivocation->invocation
...
 januar->january
 janurary->january
 Januray->January
+japaneese->japanese
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Capitalise the correction.

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -8339,22 +8339,32 @@ ivocation->invocation
...
 jaques->jacques
 javasript->javascript
 javasrript->javascript
 jave->java, have,
 jeapardy->jeopardy
+jeffies->jiffies
+jelous->jealous
+jelusey->jealousy
+jepordize->jeopardize
+jewl->jewel
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Suggest Jew?

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: You could still suggest jalousie if you want?

Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -10132,12 +10135,19 @@ javasript->javascript
 javasrript->javascript
 jave->java, have,
 jeapardy->jeopardy
+jeffies->jiffies
+jelous->jealous
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Could add jelousy->jealousy

Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

**Pull request merged**